test/parallel/test-crypto-random.js

```diff
@@ -24,8 +24,8 @@ process.setMaxListeners(256);
     {},
     []
   ].forEach(function(value) {
-    assert.throws(function() { f(value); });
-    assert.throws(function() { f(value, function() {}); });
+    assert.throws(function() { f(value); }, /size must be a number >= 0/);
+    assert.throws(function() { f(value, function() {}); }, /size must be a number >= 0/);
```

Comment:
To pass `make lint` this should be wrapped and indented to `assert.throws(` on the next line. :)